.circleci/config.yml

```diff
@@ -183,11 +183,8 @@ jobs:
           name: Run E2E tests
           command: npm run test:e2e
       - store_artifacts:
-          path: playwright-report
-          destination: playwright-report
-      - store_artifacts:
-          path: test-results
-          destination: test-results
+          path: test_results/playwright-report
+          destination: test_results/playwright-report
```

Comment:
This seems odd -- the PlayWright config points to `./e2e-tests/test_results/` for general test output (I assume videos, etc -- artefacts, effectively), and more specifically `./e2e-tests/playwright-report/` for the PlayWright report:

- https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/blob/f6d191568b9bd460e890c2c8bc942d47d59c52bc/playwright.config.ts#L7
- https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/blob/f6d191568b9bd460e890c2c8bc942d47d59c52bc/playwright.config.ts#L13

Wouldn't this only store the report, as opposed to report _and_ artefacts?